ecdsa/Cargo.toml

```diff
@@ -15,17 +15,17 @@ edition = "2021"
 rust-version = "1.56"
 
 [dependencies]
-elliptic-curve = { version = "0.11.6", default-features = false, features = ["sec1"] }
-signature = { version = ">= 1.3.1, <1.5", default-features = false, features = ["rand-preview"] }
+elliptic-curve = { git = "https://github.com/khonsulabs/traits", branch = "digest", default-features = false, features = ["sec1"] }
```

Comment:
I'd suggest using a `patch.crates-io` directive for this so you don't need to duplicate the git URL everywhere the crate is referenced

Reply:
The reason I did it like this is to make it easy for current consumers of this branch to use it without having to spam `patch`es throughout their crates. I am assuming this is no problem because it's not intended to merge this without git dependency anyway?

Reply:
I've done both `git`-based dependencies and prereleases for `elliptic-curve` in the past when a new version is under development.

If you'd like me to cut a prerelease I can, but `git` is a little more agile for development purposes.

Reply:
If cutting a pre-release for elliptic-curve is doable then that would be great, it would unblock our projects from a release on crates.io too.

Otherwise if you want to merge this it's no problem for me to remove this and replace it with `[patch]`.

Reply:
I can do that